### ISSUE TITLE
Handling bodies for raw endpoints in the dashboard and TypeScript client 

### ISSUE THREAD

**Issue by vilhelmmelkstam** [encoredev/encore#50]
This issue covers the response and request bodies of raw endpoint in the dashboard and in the TypeScript client.

### Dashboard
Currently, the even though Raw Endpoints may have request and response bodies, there are no way of editing the request body or viewing the response body. I understand that encore can't know if there will be a body and the format of it, but we should at least be able to edit it as if it were text. The same goes for viewing the response, all non-raw endpoints show their response, and the raw endpoints should probably show their reponse as well, probably just in plain-text.

 ### TypeScript Client
 A raw endpoint is currently generated into a function that looks like the following.
 ```typescript
public World(): Promise<void> {
    return this.baseClient.doVoid("POST", `/hello.World`)
}
```
However, this does not allow for neither a response nor request body. Therefore, I think it would be nice if the client would in some way handle bodies. Example solutions are:
1. Make the function take in a `ReadableStream` (The type of the fetch request body) and return the `Request` object.
2. Make the function take a `ReadableStream` and return it as a string. If a user wants it in JSON, they can parse it.
3. Assume the request and response body to still be JSON and use generics to specify the request and response schema. This would still allow for strings to be sent as they are JSON-strinfigy-able.

**Comment by eandre** [comment]
Thanks for writing this up. I agree completely. I wonder if it would make sense to add a parameter that is effectively the `fetch` options object (https://developer.mozilla.org/en-US/docs/Web/API/Request/Request -> `init`), and return the `Response`? That would allow callers to customize any behavior that the underlying `fetch` API supports.

However at some point we should also start thinking about making the generated code support both client-side and server-side JavaScript (Node/Deno). I think the easiest approach for doing this would be to create an npm package that provides the base client (`@encoredev/client` or something). We don't need to do that as part of this change (we're already assuming the existence of `fetch` in the global scope), but something to think about.

**Comment by vilhelmmelkstam** [comment]
That sounds reasonable. This would also allow developers to set other options (such as `credentials: include`, something that we currently add manually).

As for server-side fetching, I don't think it's a major problem as adding a polyfill such as [unfetch](https://github.com/developit/unfetch) is an easy fix. It's not the most elegant solution, but at least quick.

**Comment by DomBlack** [comment]
@vilhelmmelkstam this has now been resolved in #223 and will be included with the next release of the Encore CLI tool.
